uuid/idp.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Mainflux
```

Comment:
Will we call this top leve package `uuid`? It does not really implement `uuid`, but rather our IdP...

I do not know, does not bother me much, but I'd like to her form @mainflux/maintainers

Reply:
This is not an [Identity provider](https://en.wikipedia.org/wiki/Identity_provider).

Reply:
maybe we should rename it to `uuid/uuid.go` ?

Reply:
Make more sense to me @manuio